src/shared/lib/oql/accessors.ts

```diff
@@ -99,7 +99,7 @@ export default class accessors {
         if (this.alterationType(d.molecularProfileId) === 'MUTATION_EXTENDED') {
             if (d.mutationType === "fusion") {
                 return null;
-            } else if (d.aminoAcidChange && d.aminoAcidChange.toLowerCase() === "promoter") {
+            } else if (d.proteinChange && d.proteinChange.toLowerCase() === "promoter") {
```

Comment:
@jj @onursumer this is not related to the bug but i decided to change this reference to aminoAcidChange as well.  Let me know if that's right or not.

Reply:
@alisman looks right to me